config/routes.rb

```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
   mount_devise_token_auth_for 'User', at: 'auth'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
+  api_version(:module => "V1", :header => {:name => "Accept", :value => "application/vnd.mycompany.com; version=1"}) do
```

Comment:
```suggestion
  api_version(:module => "api/v1", path: { value: 'api/v1' }, defaults: { format: :json }) do
```

This is the  way we handle it in Wolox. That way you preserve the `api` in the path, and  the  header wasn't necessary

Reply:
Done